velox/exec/Task.cpp

```diff
@@ -2229,23 +2230,44 @@ ContinueFuture Task::taskCompletionFuture() {
 std::string Task::toString() const {
   std::lock_guard<std::timed_mutex> l(mutex_);
   std::stringstream out;
-  out << "{Task " << shortId(taskId_) << " (" << taskId_ << ")";
+  out << "{Task " << shortId(taskId_) << " (" << taskId_ << ")" << std::endl;
```

Comment:
We can tell from driver's reference count? Why we need to make shallow drivers? Thanks!

Reply:
@xiaoxmeng 

Thanks for taking time to look at the change!
Hard to find a reviewer these days. 🙁

We need Driver shadows for few reasons:
1. When task is terminated it loses all references to the Drivers - meaning we can't find drivers that might still linger out there.
2. If there are Driver shadows - then we can log some details about these drivers.
3. If there are Driver shadows - then we know that it is them making the Task zombie and not something else.

Let me know if that explains the idea behind them.,